maestro-ios-driver/src/main/kotlin/util/LocalSimulatorUtils.kt

```diff
@@ -334,49 +354,111 @@ object LocalSimulatorUtils {
             .map { "${it.key}=${translatePermissionValue(it.value)}" }
             .joinToString(",")
 
-        try {
-            runCommand(
-                listOf(
-                    "$homedir/.maestro/deps/applesimutils",
-                    "--byId",
-                    deviceId,
-                    "--bundle",
-                    bundleId,
-                    "--setPermissions",
-                    argument
+        if (argument.isNotEmpty()) {
```

Comment:
nit: Can you rename the variable argument here? Its difficult to understand what's happening here.